docs/marketplace/dev/front-end/ui-components-library/data-transformers/collate.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Data Transformer Collate
+description: This document provides details about the Data Transformer Collate service in the Components Library.
+template: concept-topic-template
+---
+
+
+This document provides details about the Data Transformer Collate service in the Components Library.
+
+## Overview
+
+Data Transformer Collate is an Angular Service that represents sorts, filters, and paginates data based on configuration.
+The meaning of the word `collate` is to collect, arrange and assemble in a specific order of sequence.
+
+```ts
+<spy-table
+  [config]="{
+    dataSource: {
+      type: 'table.inline',
```

Comment:
Please replace `table.inline` datasource with simple `inline` and use only `collate` data transformer in the example to simplify it.